graal/graal_bindist.bzl

```diff
@@ -220,6 +220,50 @@ def _get_platform(ctx):
     else:
         fail("Unsupported operating system: " + ctx.os.name)
 
+def _good_execute(ctx, cmd):
+    res = ctx.execute(cmd)
+    if res.return_code != 0:
+        fail("{cmd} returned {code} with stdout:\n{stdout}\n\nstderr:\n\n{stderr}".format(
+            cmd = cmd,
+            code = res.return_code,
+            stdout = res.stdout,
+            stderr = res.stderr
+        ))
+    return res.stdout
+
+def _get_hash_linux_libc(ctx):
+    res = _good_execute(ctx, ["ldd", "/bin/ls"])
+    lines = res.splitlines()
+    matched = [line for line in lines if "libc.so.6" in line]
+    if len(matched) == 1:
+        line = matched[0]
+        start = line.index(' => ') + 4
+        path_start = line[start:]
+        path = path_start[:path_start.index(' ')]
+        return _good_execute(ctx, ["shasum", path])
+    else:
+        fail("did not match one libc.so.6 line: %s" % matched)
+
+def _get_hash_mac_libc(ctx):
+    res = _good_execute(ctx, ["otool", "-L", "/bin/ls"])
+    lines = res.splitlines()
+    matched = [line for line in lines if "libSystem.B.dylib" in line]
```

Comment:
this path isn't a real path on my machine... so it claims `/usr/lib/libSystem.B.dylib` exists, but when I `ls` it, it isn't there. So I just take the full line which has some compatibility and version info and hope for the best.